pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/opplysningerbruktiberegningenalder/OpplysningerBruktIBeregningenTrygdetidTabeller.kt

```diff
@@ -0,0 +1,237 @@
+                    )
+                }
+            }
+            //norskTTAP2011Botid_001
+            showIf(alderspensjonVedVirk.erEksportberegnet and regelverkstype.isOneOf(AP2011)) {
+                paragraph {
+                    text(
+                        Bokmal to "Har du vært medlem i folketrygden i mindre enn 20 år skal trygdetiden etter kapittel 19 fastsettes til antall år med pensjonspoeng. Du har mindre enn 20 års medlemstid og trygdetiden din etter kapittel 19 er derfor fastsatt til antall år med pensjonspoeng.",
+                        Nynorsk to "Har du vore medlem i folketrygda i mindre enn 20 år skal trygdetida etter kapittel 19 fastsetjast til antal år med pensjonspoeng. Du har mindre enn 20 års medlemstid og trygdetida di etter kapitel 19 er difor fastsett til antal år med pensjonspoeng.",
+                        English to "If you have been a member of the Norwegian National Insurance Scheme for less than 20 years in total, your national insurance coverage pursuant to chapter 19 will be the same as number of years you have accumulated pensionable earnings. You have been a member for less than 20 years in total and your period of national insurance coverage pursuant to chapter 19 is the same as number of years you have accumulated pensionable earnings.",
+                    )
+                }
+            }
+
+            //norskTTAP2016Eksport_001
+            showIf(inngangOgEksportVurdering.eksportBeregnetUtenGarantipensjon_safe.ifNull(false)) {
+                paragraph {
+                    text(
+                        Bokmal to "Hvis du har mindre enn 20 års medlemstid, har du ikke rett på garantipensjon når du er bosatt i utlandet.",
+                        Nynorsk to "Dersom du har mindre enn 20 års medlemstid, har du ikkje rett på garantipensjon når du bur i utlandet.",
+                        English to "If you have been a member of the Norwegian National Insurance Scheme for less than 20 years in total, you are not eligible for any guaranteed pension when you live abroad.",
+                    )
+                }
+            }
+
+            showIf(trygdetidNorge.size().greaterThan(0)) {
+                paragraph {
+                    text(
+                        Bokmal to "Tabellen nedenfor viser perioder vi har registrert at du har bodd og/eller arbeidet i Norge. Disse opplysningene er brukt for å fastsette din norske trygdetid.",
```

Comment:
Blir dette riktig tekst? Som eg ser i metatool er vi no på norskTTTabellInnl_001, og den har tekst "Tabellen nedenfor viser perioder vi har brukt for å fastsette din norske trygdetid."?

Reply:
Var på gammel MFL. Denne har endret seg.

Reply:
Kopierte inn teksten under med uhell